dts/bindings/arm/nordic,nrf-uicr.yaml

```
@@ -0,0 +1,14 @@
+title: Nordic UICR (User Information Configuration Registers)
```

The content of this file should be like that one in [_nordic,nrf-ficr.yaml_](https://github.com/zephyrproject-rtos/zephyr/blob/5b8488a93626be2603c296f7b0e5f3a9c4049e02/dts/bindings/arm/nordic%2Cnrf-ficr.yaml#L1).

```suggestion
---
title: Nordic UICR (User Information Configuration Registers)
version: 0.1

description: >
    Binding for the Nordic UICR (User Information Configuration Registers)

properties:
    compatible:
      type: string
      category: required
      description: compatible strings
      constraint: "nordic,nrf-uicr"
      generation: define

    reg:
      type: array
      description: mmio register space
      generation: define
      category: required

...
```